src/mono/wasm/runtime/binding_support.js

```diff
@@ -1760,14 +2019,12 @@ var BindingSupportLib = {
 
 			var js_name = BINDING.conv_string (nameRoot.value);
 			if (!js_name || (typeof(js_name) !== "string")) {
-				setValue (is_exception, 1, "i32");
-				return BINDING.js_string_to_mono_string ("ERR12: Invalid method name object '" + nameRoot.value + "'");
+				return BINDING.wrap_error(is_exception, "ERR12: Invalid method name object '" + nameRoot.value + "'");
 			}
 
 			var obj = BINDING.get_js_obj (js_handle);
 			if (!obj) {
-				setValue (is_exception, 1, "i32");
-				return BINDING.js_string_to_mono_string ("ERR13: Invalid JS object handle '" + js_handle + "' while invoking '"+js_name+"'");
+				return BINDING.wrap_error(is_exception, "ERR13: Invalid JS object handle '" + js_handle + "' while invoking '"+js_name+"'");
```

Comment:
Use JS ``` `... ${js_handle} ... ${js_name}` ``` literals for stuff like this if possible

Reply:
Agreed, but let's do that on separate PR together with general `lint`